modules/installation-operators-config.adoc

```diff
@@ -25,31 +25,31 @@ Operators so that they all become available.
 $ watch -n5 oc get clusteroperators
 
 NAME                                 VERSION   AVAILABLE   PROGRESSING   DEGRADED   SINCE
-authentication                       4.2.0     True        False         False      69s
-cloud-credential                     4.2.0     True        False         False      12m
-cluster-autoscaler                   4.2.0     True        False         False      11m
-console                              4.2.0     True        False         False      46s
-dns                                  4.2.0     True        False         False      11m
-image-registry                       4.2.0     False       True          False      5m26s
-ingress                              4.2.0     True        False         False      5m36s
-kube-apiserver                       4.2.0     True        False         False      8m53s
-kube-controller-manager              4.2.0     True        False         False      7m24s
-kube-scheduler                       4.2.0     True        False         False      12m
-machine-api                          4.2.0     True        False         False      12m
-machine-config                       4.2.0     True        False         False      7m36s
-marketplace                          4.2.0     True        False         False      7m54m
-monitoring                           4.2.0     True        False         False      7h54s
-network                              4.2.0     True        False         False      5m9s
-node-tuning                          4.2.0     True        False         False      11m
-openshift-apiserver                  4.2.0     True        False         False      11m
-openshift-controller-manager         4.2.0     True        False         False      5m943s
-openshift-samples                    4.2.0     True        False         False      3m55s
-operator-lifecycle-manager           4.2.0     True        False         False      11m
-operator-lifecycle-manager-catalog   4.2.0     True        False         False      11m
-service-ca                           4.2.0     True        False         False      11m
-service-catalog-apiserver            4.2.0     True        False         False      5m26s
-service-catalog-controller-manager   4.2.0     True        False         False      5m25s
-storage                              4.2.0     True        False         False      5m30s
+authentication                       4.3.0     True        False         False      69s
+cloud-credential                     4.3.0     True        False         False      12m
+cluster-autoscaler                   4.3.0     True        False         False      11m
+console                              4.3.0     True        False         False      46s
+dns                                  4.3.0     True        False         False      11m
+image-registry                       4.3.0     True        False         False      5m26s
```

Comment:
Bumped the version for all of the entries (FYI @kalexand-rh). Need to verify the image-registry update is valid for all platforms, not just bare metal and vSphere.